src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -21,35 +21,35 @@ You can customize the creation of a new [BrowserContext](/javascript-api/xk6-bro
+| <BWIPT id="436"/> hasTouch          | boolean | `false`         | Whether to simulate a device with touch events.                                                                                                                                                                                                   |
+| httpCredentials                     | object  | `null`          | Sets the credentials for HTTP authentication using Basic Auth. See: [httpCredentials](#httpcredentials-options).                                                                                                                                  |
+| ignoreHTTPSErrors                   | boolean | `false`         | Whether to ignore HTTPS errors that may be caused by invalid certificates.                                                                                                                                                                        |
+| isMobile                            | boolean | `false`         | Whether to simulate a mobile device.                                                                                                                                                                                                              |
+| javaScriptEnabled                   | boolean | `true`          | Whether to activate JavaScript support for the context.                                                                                                                                                                                           |
+| locale                              | string  | system          | Specifies the user's locale, such as `en-US`, `tr-TR`, etc.                                                                                                                                                                                       |
+| offline                             | boolean | `false`         | Whether to emulate an offline network.                                                                                                                                                                                                            |
+| permissions                         | Array   | `null`          | Permissions to grant for the context's pages. See [browserContext.grantPermissions()](/javascript-api/xk6-browser/browsercontext#browsercontext-grantpermissions-permissions-options) for the options.                                            |
+| reducedMotion                       | string  | `no-preference` | Minimizes the amount of motion by emulating the 'prefers-reduced-motion' media feature. It can be one of `'reduce'` and `'no-preference'`. See [page.emulateMedia()](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options. |
```

Comment:
```suggestion
| reducedMotion                       | string  | `'no-preference'` | Minimizes the amount of motion by emulating the 'prefers-reduced-motion' media feature. It can be one of `'reduce'` and `'no-preference'`. See [page.emulateMedia()](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options. |
```

Same for other instances.